<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="logButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Log</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModes">
    <property name="title">
     <string>Modes</string>
    </property>
    <addaction name="actionTimeView"/>
    <addaction name="actionXyView"/>
    <addaction name="actionExerciseView"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAutoRange"/>
    <addaction name="actionLimits"/>
    <addaction name="actionThresholds"/>
    <addaction name="separator"/>
    <addaction name="actionSingles"/>
    <addaction name="separator"/>
    <addaction name="action2dChannels"/>
    <addaction name="separator"/>
    <addaction name="actionExThresholds"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpfile"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionLangEn"/>
     <addaction name="actionLangDe"/>
    </widget>
    <addaction name="actionChangeDefaults"/>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuModes"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTimeView">
   <property name="text">
    <string>Time View</string>
   </property>
  </action>
  <action name="actionXyView">
   <property name="text">
    <string>XY View</string>
   </property>
  </action>
  <action name="actionExerciseView">
   <property name="text">
    <string>Exercise View</string>
   </property>
  </action>
  <action name="actionLimits">
   <property name="text">
    <string>Limits</string>
   </property>
  </action>
  <action name="actionen">
   <property name="text">
    <string>en</string>
   </property>
  </action>
  <action name="actionde">
   <property name="text">
    <string>de</string>
   </property>
  </action>
  <action name="action2dChannels">
   <property name="text">
    <string>2D-Channels</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Jansenberger">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelpfile">
   <property name="text">
    <string>Help-file</string>
   </property>
  </action>
  <action name="actionChange_Defaults">
   <property name="text">
    <string>Change Defaults</string>
   </property>
  </action>
  <action name="actionExThresholds">
   <property name="text">
    <string>Exercise Thresholds</string>
   </property>
  </action>
  <action name="actionSensor">
   <property name="text">
    <string>Views</string>
   </property>
  </action>
  <action name="actionSingles">
   <property name="text">
    <string>Singles</string>
   </property>
  </action>
  <action name="actionThresholds">
   <property name="text">
    <string>Thresholds</string>
   </property>
  </action>
  <action name="actionAutoRange">
   <property name="text">
    <string>AutoRange</string>
   </property>
  </action>
  <action name="actionChangeDefaults">
   <property name="text">
    <string>Change Defaults</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionLangEn">
   <property name="text">
    <string>en</string>
   </property>
  </action>
  <action name="actionLangDe">
   <property name="text">
    <string>de</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
